src/js/controllers/onboarding/collectEmail.js

```diff
@@ -2,10 +2,10 @@
 
 angular.module('copayApp.controllers').controller('collectEmailController', function($scope, $state, $log, $timeout, $http, $httpParamSerializer, $ionicConfig, profileService, configService, walletService, appConfigService, emailService) {
 
-  var wallet, walletId;
+  var wallet, walletId,coinAddress;
```

Comment:
Like I said below, I would probably move the `walletService.getAddress` function into the `collectEmail` function.